articles/user-guide/index.md

```diff
@@ -48,7 +48,7 @@ In the different topics of this guide, we detail the core concepts of the Q# lan
 
 ### Quantum Simulators and Resource Estimators
 
-- [Quantum simulators and host applications](xref:microsoft.quantum.machines): An overview of the different simulators available, as well as the general execution model between host programs and target machines.
+- [Quantum simulators and host applications](xref:microsoft.quantum.machines): An overview of the different simulators available, as well as the general interaction model between host programs and target machines.
```

Comment:
I'm not sure that the term "interaction model" works as a substitute in this case; it looks like in some cases, "execution model" is still used (e.g.: https://docs.microsoft.com/en-us/windows/win32/opengl/execution-model). If that's not in keeping with current guidance, however, I'd suggest reworking this bullet item to avoid the term altogether. Perhaps something like the following?

> An overview of the different simulators available, as well of how host programs and target machines work together to run Q# programs.

Reply:
Yeah, that's better!